scylla_test.go

```diff
@@ -112,15 +112,17 @@ func TestScyllaConnPickerRemove(t *testing.T) {
 	}
 }
 
-func mockConn(shard string) *Conn {
+func mockConn(shard int) *Conn {
 	return &Conn{
 		supported: map[string][]string{
-			"SCYLLA_SHARD":               {shard},
-			"SCYLLA_NR_SHARDS":           {"4"},
-			"SCYLLA_SHARDING_IGNORE_MSB": {"12"},
 			"SCYLLA_PARTITIONER":         {"org.apache.cassandra.dht.Murmur3Partitioner"},
```

Comment:
This should be also part of `scyllaSupported`

Reply:
Done. Force-pushed updated version.